app/views/games/show.html.erb

```diff
@@ -1,3 +1,19 @@
-<h3 class='col-12 offset-4'><%= @game.name %></h3>
+<h3 class='col-10 offset-5'><%= @game.name %></h3>
+<div class='col-10 offset-3'>
+    <div class="chessboard">
+        <% 0.upto(7) do |y| %>
+            <% 0.upto(7) do |x| %>
+                <% if (y+x).even? %>

```

Comment:
Pasting here one of the previous comments regarding white/black


Let's do another small refactoring here:

We can add a method to games_helper.rb sth like:
```
def black_whit_cell(x,y)
   (y+x).even? ? "white" : "black"
end

```
so that we can skip the whole if/else block here and just have a div <div class="#{black_white_cell(x,y)}"></div>